pennylane/ops/op_math/exp.py

```diff
@@ -234,7 +234,10 @@ def eigvals(self):
         return qml.math.exp(self.coeff * base_eigvals)
 
     def label(self, decimals=None, base_label=None, cache=None):
-        return base_label or "Exp"
+        coeff = (
+            self.coeff if decimals is None else format(math.toarray(self.coeff), f".{decimals}f")
```

Comment:
taken from [Operator.label()](https://github.com/PennyLaneAI/pennylane/blob/b025e1d0fcf30a69360eb59971ad9cc7a48e7d05/pennylane/operation.py#L723). Note that `decimals=None` will result in a complex coeff being wrapped in parentheses, while providing a decimal will invoke formatting which removes it. No biggie, but thought I'd point it out. Tests reflect this.